evaluation_criteria.md

```diff
@@ -1,56 +1,63 @@
-# Evaluation Criteria
+# Your Name Evaluation
```

Comment:
I the idea that this would up looking like this?

```
# Ibraheem Craft's Evaluation
```

If so, I wonder if it would be clear if it were something like

```markdown
# <!-- Your Name -->'s Evaluation
```

Or maybe use some other brackets to indicate that the user-author should manually insert a value? (I don't think we have a set convention for this...)